uaclient/tests/test_cli.py

```diff
@@ -779,179 +800,81 @@ def test_argparse_errors_well_formatted(
-        logging.info("after setup")
-
-        assert not log_file.exists()
-
-    @pytest.mark.parametrize("log_filename", (None, "file.log"))
-    @mock.patch("uaclient.cli.config")
-    def test_file_log_configured_if_root(
-        self,
-        m_config,
-        log_filename,
-        logging_sandbox,
-        tmpdir,
-    ):
-        if log_filename is None:
-            log_filename = "default.log"
-            log_file = tmpdir.join(log_filename)
-            m_config.CONFIG_DEFAULTS = {"log_file": log_file.strpath}
-        else:
-            log_file = tmpdir.join(log_filename)
-
-        setup_logging(logging.INFO, logging.INFO, log_file=log_file.strpath)
-        logging.info("after setup")
-
-        assert "after setup" in log_file.read()
-
-    def test_file_log_configured_if_already_present(
-        self,
-        logging_sandbox,
-        tmpdir,
-    ):
-        some_file = tmpdir.join("default.log")
-        logging.getLogger().addHandler(logging.FileHandler(some_file.strpath))
-
-        log_file = tmpdir.join("file.log")
-
-        logging.error("before setup")
-        setup_logging(logging.INFO, logging.INFO, log_file=log_file.strpath)
-        logging.error("after setup")
-
-        content = log_file.read()
-        assert re.match(r'\[.*"ERROR", "before setup"', content) is None
-        assert re.match(r'\[.*"ERROR",.*"after setup"', content)
+        log_message = "Logging to Console"
+        with mock.patch(
+            "uaclient.cli.config.UAConfig", return_value=FakeConfig()
+        ):
+            setup_logging(logging.INFO, logging.DEBUG)
+        logging.log(logging.INFO, "Logging to Console")
+        out, err = capsys.readouterr()
+        assert out == ""
+        assert log_message == err.strip()
 
-    @mock.patch("uaclient.cli.config.UAConfig")
-    def test_custom_logger_configuration(
+    def test_correct_handlers_added_to_logger(
         self,
-        m_config,
         logging_sandbox,
-        tmpdir,
         FakeConfig,
+        get_logger_handler,
     ):
-        log_file = tmpdir.join("file.log")
-        cfg = FakeConfig({"log_file": log_file.strpath})
-        m_config.return_value = cfg
-
-        custom_logger = logging.getLogger("for-my-special-module")
-        root_logger = logging.getLogger()
-        n_root_handlers = len(root_logger.handlers)
-
-        setup_logging(logging.INFO, logging.INFO, logger=custom_logger)
-
-        assert len(custom_logger.handlers) == 2
-        assert len(root_logger.handlers) == n_root_handlers
+        console_level = logging.INFO
+        log_level = logging.DEBUG
+        logger = logging.getLogger("logger_a")
+        with mock.patch(
+            "uaclient.cli.config.UAConfig", return_value=FakeConfig()
+        ):
+            setup_logging(console_level, log_level, logger=logger)
+        assert len(logger.handlers) == 2
+        assert logger.handlers == get_logger_handler(
+            logger, console_level, log_level
+        )
```

Comment:
Instead of these assertions, could we do something more like this?
```
assert repr(logger.handlers) == repr([logging.StreamHandler(sys.stderr), logging.FileHandler(log_file)])
```
(IDK if that will work, but something like that would be nice)

Reply:
Quick question, is there a specific reason for this specific test?

Reply:
Looking closer, this particular test is probably not needed because the code it is testing is so simple IMO

Reply:
After discussion, we saw it is better to check the types of the handlers and the number of the handlers.

Reply:
We could combine with the `test_previous_handlers_removed_in_logger` method.